From e424a47728a6d685098f2b7acee6478b0e973703 Mon Sep 17 00:00:00 2001
From: "Voleti, Murali" <murali.voleti@intel.com>
Date: Fri, 13 Jul 2018 10:33:04 +0200
Subject: [PATCH] Merge pull request #69 in SW_UGW/linux from
 UGW_SW-26936-klocwork-fixes-on-8.x to master

* commit 'b28ecb8bbf78f89daaf94c106fbb2df0a5d76896':
  In PPA file ppa_stack_al.h Function prototype changed
---
 include/net/ppa/ppa_stack_al.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/net/ppa/ppa_stack_al.h b/include/net/ppa/ppa_stack_al.h
index 15dbf2dc08d6..fd1f584af1cc 100644
--- a/include/net/ppa/ppa_stack_al.h
+++ b/include/net/ppa/ppa_stack_al.h
@@ -1857,7 +1857,7 @@ void *ppa_mem_cache_alloc(PPA_MEM_CACHE *p_cache);
   - PPA_FAILURE, if the memory cache free operation fails. \n
   \note
  */
-void ppa_mem_cache_free(void *buf, PPA_MEM_CACHE *p_cache);
+int32_t ppa_mem_cache_free(void *buf, PPA_MEM_CACHE *p_cache);
 /*! \brief This function does a byte copy from source buffer to destination buffer for the specified number of bytes.
   \param[in] dst Pointer to destination buffer to copy to.
   \param[in] src Pointer to source buffer to copy from.
